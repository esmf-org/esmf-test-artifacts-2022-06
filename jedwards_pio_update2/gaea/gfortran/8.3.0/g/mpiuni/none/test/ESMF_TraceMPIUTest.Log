build time -- 2022-03-07 23:49:55
20220308 003552.636 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220308 003552.636 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220308 003552.636 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220308 003552.636 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220308 003552.636 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220308 003552.636 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220308 003552.636 INFO             PET0 Running with ESMF Version   : v8.3.0b08-93-g7b747143aa
20220308 003552.636 INFO             PET0 ESMF library build date/time: "Mar  7 2022" "23:48:07"
20220308 003552.636 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpiuni_g_jedwards_pio_update2
20220308 003552.636 INFO             PET0 ESMF_COMM                   : mpiuni
20220308 003552.638 INFO             PET0 ESMF_MOAB                   : enabled
20220308 003552.638 INFO             PET0 ESMF_LAPACK                 : enabled
20220308 003552.638 INFO             PET0 ESMF_NETCDF                 : enabled
20220308 003552.638 INFO             PET0 ESMF_PNETCDF                : disabled
20220308 003552.638 INFO             PET0 ESMF_PIO                    : disabled
20220308 003552.638 INFO             PET0 ESMF_YAMLCPP                : enabled
20220308 003552.638 INFO             PET0 ESMF Tracing Enabled
20220308 003552.638 INFO             PET0 ESMF Profiling Enabled
20220308 003552.638 INFO             PET0 ESMF Trace/Profile clock: REALTIME
20220308 003552.647 WARNING          PET0 ESMF Tracing/Profiling could not load dynamic instrumentation functions.
20220308 003552.647 INFO             PET0  Beginning Test, file ESMF_TraceMPIUTest.F90, line 75
20220308 003552.647 INFO             PET0  NUMBER_OF_PROCESSORS           1
20220308 003552.647 INFO             PET0  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 97
20220308 003552.647 INFO             PET0  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 116
20220308 003552.647 INFO             PET0  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 127
20220308 003552.648 INFO             PET0  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 140
20220308 003552.648 INFO             PET0  PASS  Test trace user region exit, ESMF_TraceMPIUTest.F90, line 149
20220308 003552.648 INFO             PET0  PASS  Test trace memory usage, ESMF_TraceMPIUTest.F90, line 157
20220308 003552.648 INFO             PET0 **************** Region Timings *******************
20220308 003552.648 INFO             PET0 Region              Count  Total (s)   Self (s)    Mean (s)    Min (s)     Max (s)
20220308 003552.648 INFO             PET0   reg1              1      0.0001      0.0001      0.0001      0.0001      0.0001
20220308 003552.659 INFO             PET0  PASS  Close trace, ESMF_TraceMPIUTest.F90, line 168
20220308 003557.659 INFO             PET0  Ending Test, file ESMF_TraceMPIUTest.F90, line 180
20220308 003557.659 INFO             PET0 Finalizing ESMF
20220308 003557.659 INFO             PET0 ESMCI_IO_Handler.C:356 ESMCI::IO_Handler::finalize() before return, localrc = 0
