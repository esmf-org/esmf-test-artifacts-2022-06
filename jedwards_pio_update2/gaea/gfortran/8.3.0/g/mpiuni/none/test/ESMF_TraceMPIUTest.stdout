build time -- 2022-03-08 23:39:32
Wed Mar  9 00:23:08 2022: [unset]:_pmi_alps_init:alps_get_placement_info returned with error -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_init:_pmi_alps_init returned -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_alps_init:alps_get_placement_info returned with error -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_init:_pmi_alps_init returned -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_alps_init:alps_get_placement_info returned with error -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_init:_pmi_alps_init returned -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_alps_init:alps_get_placement_info returned with error -1
Wed Mar  9 00:23:08 2022: [unset]:_pmi_init:_pmi_alps_init returned -1
  Beginning Test, file ESMF_TraceMPIUTest.F90, line 75
  NUMBER_OF_PROCESSORS           1
  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 97
  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 116
 MPI_ALLREDUCE returned           -1
  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 127
  PASS  Test trace user region enter, ESMF_TraceMPIUTest.F90, line 140
  PASS  Test trace user region exit, ESMF_TraceMPIUTest.F90, line 149
  PASS  Test trace memory usage, ESMF_TraceMPIUTest.F90, line 157
  PASS  Close trace, ESMF_TraceMPIUTest.F90, line 168
  Ending Test, file ESMF_TraceMPIUTest.F90, line 180
  PET           0  Test Elapsed Time    5000.3309999999992       msec.
